src/main/java/drukier/maze/MazeCell.java

```diff
@@ -0,0 +1,38 @@
+package drukier.maze;
+
+public class MazeCell {
+
+    private int cellY;
+    private int cellX;
+
+    private boolean visited = false;
+
+    public MazeCell(int cellX, int cellY){
+        this.cellY = cellX;
+        this.cellX = cellY;
+    }
+
+    public boolean isVisited() {
+        return visited;
+    }
+
+    public void setVisited(boolean visited) {
+        this.visited = visited;
+    }
+
+    public int getCellY() {
+        return cellY;
+    }
+
+    public void setCellY(int cellY) {
```

Comment:
don't create methods you aren't actually using.
In this instance you're never going to set the cell's XY coordinate to something else.